config/7.12.0/pim/process-migration.yaml

```diff
@@ -43,15 +43,18 @@ processMigration:
                 imagePullPolicy: Always
                 resources:
                   limits:
-                  memory: 512Mi
+                    memory: 512Mi
                 ports:
                   - name: http
                     containerPort: 8080
                     protocol: TCP
+                  - name: https
```

Comment:
PIM is not starting with tls so this port shouldn't be exposed

Reply:
already removed on local changes.